experimental/GaloisGrp/GaloisGrp.jl

```diff
@@ -2135,6 +2184,21 @@ function cauchy_ideal(f::fmpz_poly; parent::MPolyRing = PolynomialRing(QQ, degre
   return cauchy_ideal(f(gen(Hecke.Globals.Qx)), parent=parent)
 end
 
+@doc Markdown.doc"""
+    cauchy_ideal(f::PolyElem{<:FieldElem})
+
+The coefficients of `f` are the elementary symmetric functions evaluated
+at the roots of `f`. The `cauchy_ideal` is the ideal generated
+by the differences between the elementary symmetric functions and the
+coefficients.
+
+# Example
+```jldoctest
+julia> Qx, x = QQ["x"]
+julia> cauchy_ideal(x^4-2)
```

Comment:
```suggestion
julia> Qx, x = QQ["x"]

julia> cauchy_ideal(x^4-2)
```

Reply:
etc. I could just push a fix, but I'll rather discuss it once we can talk to avoid conflicts